src/IssueCreator/Dialogs/ManageIssue.cs

```diff
@@ -1,26 +1,21 @@
 ﻿using IssueCreator.Helpers;
 using IssueCreator.Logging;
 using IssueCreator.Models;
-using Octokit;
 using System;
-using System.Collections.Generic;
 using System.ComponentModel;
-using System.Data;
 using System.Diagnostics;
-using System.Drawing;
-using System.Linq;
-using System.Text;
 using System.Threading.Tasks;
 using System.Windows.Forms;
 
 namespace IssueCreator.Dialogs
 {
     public partial class ManageIssue : Form
-    {
-        private IssueManager _issueManager;
-        private FileLogger _logger;
-        private Settings _settings;
-
+    {
+        protected IssueManager _issueManager;
+        protected FileLogger _logger;
+        protected Settings _settings;
+        protected IssueObject LastIssueLoaded { get; private set; }

```

Comment:
I would move this to the IssueManager object. Would that work?

Reply:
Yeah, I like that better.